File: Coding Ninjas - CodeStudio/Basics Of C++ Guided Path/sum_of_even_&_odd.cpp
#include <iostream>
using namespace std;

int main() {
	int n, sumEven = 0, sumOdd = 0;
    cin >> n;
    while(n > 0) {
        int d = n % 10;
        n /= 10;
        if(d % 2 == 0)
            sumEven += d;
        else
            sumOdd += d;
    }
    cout << sumEven << " " << sumOdd << endl;
    return 0;
}

File: Coding Ninjas - CodeStudio/Basics of Java Guided Path/sum_of_even_&_odd.java
import java.util.Scanner;
class Solution {
	
	public static void main(String args[]) {
		
		Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        int sumEven = 0, sumOdd = 0;
        
        while(n > 0) {
            int d = n % 10;
            n /= 10;
            
            if(d % 2 == 0)
                sumEven += d;
            else
                sumOdd += d;
        }
        
        System.out.println(sumEven + " " + sumOdd);
		
	}
}